.gitignore

```diff
@@ -0,0 +1,26 @@
+ # General
```

Comment:
Missing the .idea which PHPStorm is probably creating

Reply:
I've added .idea